xbmc/interfaces/legacy/Player.cpp

```diff
@@ -417,6 +423,20 @@ namespace XBMCAddon
       CServiceBroker::GetGUI()->GetWindowManager().SendMessage(msg);
     }
 
+    InfoTagGame* Player::getGameInfoTag()
+    {
+      XBMC_TRACE;
+      if (!getAppPlayer()->IsPlayingGame())
+        throw PlayerException("XBMC is not playing any game file");
```

Comment:
Probably don't want to hardcode XBMC into even a throw message

Reply:
Fixed by renaming to Kodi, as it appears in the function above.